contracts/Mutants.sol

```diff
@@ -257,6 +259,7 @@ contract Mutants is IMutants, ERC721, Ownable, ReentrancyGuard, VRFConsumerBase
         require(tokenIds.length <= CLAIM_LIMIT, "Exceeds claim limit");
 
         for (uint256 i = 0; i < tokenIds.length; i++) {
+            if (_msgSender() != ownerOf(tokenIds[i])) revert Mutant_SenderNotTokenOwner();
```

Comment:
same applies here as with Scales PR. `transferFrom` will not allow anyone but the owner to call this function